Products/Zuul/facades/metricfacade.py

```diff
@@ -51,24 +53,144 @@ def _isRunningFromUI(context):
     return safe_hasattr(context.REQUEST, 'SESSION')
 
 
+class MetricConnection(object):
+
```

Comment:
Please remove this blank line.  Doc comments, stylistically, begin on the line following the declaration they document.